.github/workflows/rust.yml

```diff
@@ -158,32 +154,6 @@ jobs:
           override: true
           components: clippy
 
-      - name: Cache dependencies
```

Comment:
Can we revert caching for the CI?

Reply:
Yes, I have like 3 commits to revert

Reply:
Caching is back https://github.com/hashintel/hash/pull/414/commits/b864096209ae578f80284487a373224e27e77546